notary-server/Dockerfile

```diff
@@ -1,25 +1,28 @@
-FROM golang:1.10.3-alpine
+FROM alpine:3.8
```

Comment:
I think its better to use eg `golang/1.10.5-alpine3.8` so we pin the Go version and the Alpine version, rather than installing the Go that Alpine happens to have at present.

Reply:
By doing so you will increase the size of the image from 21MB to 268MB.

Reply:
We could do 'apk add go=1.10.5' if you'd prefer?

Reply:
Ah ugh yes. I am just trying to get Alpine updated with the latest Go 1.10 patch release, as it is currently only on 1.10.1.

Reply:
That would be a better solution.

Although it doesn't seem to affect Notary.  It tested just fine with Apline 3.8.

Reply:
So when installing *this* PR I see that Apline 3.8 installs Go version `1.10.5-r0`.

```(19/22) Installing go (1.10.5-r0)```

Is that not adequate?

Reply:
Yes, Its fixed now, I got an updated pushed yesterday, all good...